README.md

```diff
@@ -225,6 +225,17 @@ This file defines your job search parameters and bot behavior. Each section cont
       - Data Scientist
     ```
 
+- `job_description_keywords:`
+  - List keywords to look for in the job description, for example, "chromium browser"
+  - Only use this field to find jobs that require specific technologies, products, or skills that is not mentioned in the job title.
+  - Warning: Any jobs without matching keywords in the job description will be skipped.
+  - Example:
+
+    ```yaml
+    job_description_keywords:
```

Comment:
I am not sure _whitelist prefix is a good idea. Usually whitelist is intended to revert the action of a larger blacklist, which doesn't quite fit this case.

Reply:
I am not sure whitelist means that. If there's a list of whitelisted items, it means these are the `only` ones allowed/accepted/approved. 

In the context of this project, it means the job description must have at least one of the listed keywords if it's to be considered a potential job match - which is what whitelisting does. The addition of the suffix is for clarity and make sure everyone understands it works as a whitelist - it will look for the presence of these keywords and reject those that do not contain any of the keywords.

Reply:
Done.